Tutorials/intro-to-octave/README.md

```diff
@@ -29,13 +49,13 @@ Let's talk about each function separately:
       2. If you want to use an external clock with 100MHz set `external_clock='100MHz'`
       3. If you want to use an external clock with 1GHz set `external_clock='1GHz'`
    2. Sets the up-converters modules.
-      1. LO  - The default LO source is internal. If you want to use an external LO you need to enter `set_octave.py` file and change the LO to the relevant external one (LO1, LO2, LO3, LO4 or LO5).
-      2. Gain - The default is zero gain. You can change it by entering `set_octave.py` file and modify the relevant command.
-      3. Trigger - The default is on. You can change it by entering `set_octave.py` file and modify the relevant command.
-   3. Sets the down-converters modules.
+      1. LO  - The default LO source is internal. If you want to use an external LO you need to enter `set_octave.py` file and change the LO to the relevant external one (LO1, LO2, LO3, LO4 or LO5)
+      2. Gain - The default is zero gain. You can change it by entering `set_octave.py` file and modify the relevant command
+      3. Trigger - The default is on. You can change it by entering `set_octave.py` file and modify the relevant command
```

Comment:
Maybe recall the different option and valid range here as well so that the customers can easily find it

Reply:
I added the options for the trigger but the gain might change after Nissim's calibration so I prefer to wait.. Now it's not really the one that is written in the documentation and Yuval is aware of it..